src/Resources/config/form.xml

```diff
@@ -2,6 +2,7 @@
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
         <service id="sonata.user.form.gender_list" class="Sonata\UserBundle\Form\Type\UserGenderListType">
+            <deprecated>The "%service_id%" service is deprecated since 4.x and will be removed in 5.0. Use choice type instead</deprecated>
```

Comment:
Please use FQCN for choice type

Reply:
Fixed, not using the full message since symfony does the same for its classes: 

https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Security/Core/SecurityContext.php#L14

vs

https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/SecurityBundle/Resources/config/security.xml#L57

Hope you agree